### ISSUE TITLE
cap formatter conflicts with base alphabet letter `c`

### ISSUE THREAD

**Issue by ykuksenko** [knausj85/knausj_talon#1164]
I noticed that the `cap` formatter conflicts with the base alphabet letter `c`. Should the formatter be renamed?

**Comment by rntz** [comment]
`escape cap` should work for inserting the word "cap" in dictation mode. In mixed mode, if you want to press `c`, you can probably say `press cap`, but I agree this is annoying. @wolfmanstout thoughts?

**Comment by knausj85** [comment]
Dumb question: do we want to worry about mixed mode currently? It is a beta-only feature, and I sorta hesitate to change the grammar for the folks currently on public talon. Perhaps this is best left to folks forks for the time being?

**Comment by rntz** [comment]
Yeah, I would also hesitate to change `cap` here without some consideration. However, when possible it would be nice to avoid things that cause problems in mixed mode, because it seems very popular and useful for those on the beta, so it may eventually hit public in some form. It is also possibly annoying that bare "cap" does not work in dictation mode? How does Dragon work here? Can you really not say "cap" in a sentence, eg "I took off my cap period" / "His cap blew away period"?

**Comment by knausj85** [comment]
Sounds reasonable. 

Using cap for capitalizing words in dictation mode was actually borrowed from dragon.

**Comment by wolfmanstout** [comment]
Thanks for flagging this, I never would have noticed because I use a different alphabet (that preceded Talon). Dragon does have the exact issue described here, so it is consistent with Dragon in that sense.

I guess we could change the default to something like "cap next"? I am personally going to keep mine the way it is because there is no conflict.

**Comment by ykuksenko** [comment]
To clarify my perspective:
- The default/base alphabet that was tuned for speed having letters in conflict is my main concern.
  - If I have to spell out something it is generally an exception. Having to worry about basic letters not working makes it a bit jarring.

- I am not really concerned about the word `cap` being in conflict. To me, it is expected that I would need to be prefix/escape various keywords with `word` to avoid clashes occasionally.

**Comment by nriley** [comment]
I'm having trouble replicating your issue. I am able to do the following in mixed mode:

* say the word cap individually which inserts a lowercase c
* say the word cap prior to a word to capitalize the next word
* say "word cap" to insert the word cap

Can you give an example of an input which in mixed mode does not produce the expected results?
